src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AotTests.cs

```diff
@@ -7,13 +7,45 @@
 using Mono.Cecil;
 using NUnit.Framework;
 using Xamarin.ProjectTools;
+using Xamarin.Android.Build;
 
 namespace Xamarin.Android.Build.Tests
 {
 	[Category ("Node-2"), Category ("AOT")]
 	[Parallelizable (ParallelScope.Children)]
 	public class AotTests : BaseTest
 	{
+		[OneTimeSetUp]
+		public void Setup ()
+		{
+			if (!IsWindows)
+				return;
+
+			var sdkPath = AndroidSdkPath;
+			var ndkPath = AndroidNdkPath;
+
+			var path = Path.Combine (Root, "temp", string.Format ("SDK Ümläüts"));
```

Comment:
Should this instead become a field, so that you don't have to copy the `Path.Combine()` logic between `Setup()` and `TearDown()`?

Reply:
Probably yes. I'll wait for the test run to complete though.